src/main/resources/META-INF/plugin.xml

```diff
@@ -206,6 +206,11 @@
                     icon="com.virtuslab.gitmachete.frontend.icons.MacheteIcons.HELP">
                 <override-text place="GitMacheteToolbar"/>
             </action>
+
+            <action id="GitMachete.TraverseAction"
```

Comment:
we need an icon for this one. 

Do you have any propositions of how it should look like? (cc @PawelLipski)